frontend/src/metabase/query_builder/components/view/NativeQueryPreviewSidebar/NativeQueryPreviewSidebar.tsx

```diff
@@ -85,7 +87,7 @@ export const NativeQueryPreviewSidebar = (): JSX.Element => {
       <Box
         style={{ flex: 1, borderTop: borderStyle, borderBottom: borderStyle }}
       >
-        {isLoading && <DelayedLoadingSpinner delay={1000} />}
+        {showLoader && <DelayedLoadingSpinner delay={1000} />}
```

Comment:
_This comment is for line 117 but GH won't allow me to comment there_

It's weird that the button to convert the query disappears when preview is loading or the loading failed.
We don't need to do this, the button should be always shown.

Reply:
But we need to make it disabled, at least.
You don't want someone clicking on that button when there's no query, or there's an error. It would just result in another error and weird UX.

Reply:
You're right, I didn't realize we used the output of preview endpoint to start a new question.

Anyway, disabling the button sounds better than not rendering it (which means having a layout shift and displaying an empty footer) :+1:

Reply:
@AlessioLaiso just to get a final confirmation - how does disabling the convert button when there's no query or when there's an error sound to you?

Reply:
Yes, disabling it is fine!